config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  root 'greetings#random_greeting'
+  get '/*path', to: 'home#index', constraints: lambda { |req| !req.xhr? && req.format.html? }
```

Comment:
Great work so far, but there are still some changes that need to be made. You seem to have made changes to your code based on the previous reviewer's feedback.

- [ ] But after the review, you have changed your router configuration, but the home page is not displaying anything and even the greeting API is not returning the proper JSON output. I would recommend fixing this issue.
![image](https://github.com/Omwomo/hello-rails-react/assets/55840999/2705f064-c852-4770-8287-54bc7b1cce87)

- [ ] Additionally, I noticed that you have addressed the previous reviewer's comments. Still, I wasn't able to see the output, so it's not possible to verify whether you have met those requirements or not. The next reviewer will also check those requests again.

Reply:
Hello @shahadat3669  when you check my README file you've first to run `bundle install` and `yarn install `commands first. Then run `yarn run build` . After that is when you run the `rails s` comaand to start the rails server and then run `yarn start` command to start the front-end application on port 8080.
The current port you are on is for the backend, if you want to view the random greetings it's on port 8080.


My application is working fine, and I made the changes requested by the previous reviewer. Here is a screenshont of the front-end application on port 8080

![Screenshot (1466)](https://github.com/Omwomo/hello-rails-react/assets/115637810/3215fb29-582e-4b62-bed6-92568547ff48)

Reply:
Hi @Omwomo, I understand that you suggested having the backend and frontend operate on different ports. However, The project objective is to run a unified application. To achieve this, you should use React for the front end to seamlessly render the HTML within the same application. In the previous Blog app, you used ERB for this purpose, but this project required to use The React instant of ERB. Therefore, I recommend that you check the documentation again and reconfigure your application accordingly. Let me know if you need any further assistance with this.